lib/fathead/git_manual/parse.py

```diff
@@ -16,24 +16,27 @@ class Command(object):
     def __init__(self, name, description, filename):
         '''Instantiate the information about the command'''
         self.name = name
-        self.description = description
+        self.description = description.replace('\n', '\\n').replace('\t', '    ')
         self.filename = filename
         self.usage = ''
 
-    def __str__(self):
+    def basic_usage(self):
         '''
         Output the git command information in the proper format required for
         DuckDuckGo Fatheads
         '''
-
         # Clean up the usage statement which can have newline characters and
         # tab characters, which mess up how it renders
         usage_cleaned = self.usage.replace('\n', '\\n').replace('\t', '    ')
 
-        # Make the abstract have the description as well as a code block
-        # for the usage of the command
-        abstract = '{}\n<pre><code>{}</pre></code>'.format(self.description,
-                                                           usage_cleaned)
+        if usage_cleaned:
+            # Make the abstract have the description as well as a code block
```

Comment:
@samskeller now that we have Fathead template elements, we should be using them for this IA.

Please make sure the `abstract` is wrapped in a `<section class="prog__container">` tag, and that the `description` text is wrapped in a `<p>`

This will help to space out all the content so it's a bit easier to read 👍 

More info here: https://docs.duckduckhack.com/resources/fathead-overview.html#formatting-the-abstract

Note: It says to use a `div` tag, it should actually be a `section` -- I've pushed a change to the docs and am waiting for them to be redeployed 👍